pkg/controller/job/jobcontroller.go

```diff
@@ -135,39 +117,26 @@ func NewJobController(podInformer cache.SharedIndexInformer, kubeClient clientse
 	return jm
 }
 
-func NewJobControllerFromClient(kubeClient clientset.Interface, resyncPeriod controller.ResyncPeriodFunc) *JobController {
```

Comment:
I realized after I moved this to jobcontroller_test.go that maybe someone outside of kube could potentially be using this. Is it ok to unexport/move or do you all want it to remain (it's only used by test code in kube)? @soltysh